lib/buy/dfx/dfx_buy_provider.dart

```diff
@@ -312,9 +312,9 @@ class DFXBuyProvider extends BuyProvider {
       final uri = Uri.https('services.dfx.swiss', actionType, {
         'session': accessToken,
         'lang': 'en',
-        'asset-out': quote.destinationCurrency.toString(),
+        'asset-out': quote.cryptoCurrency.toString(),
         'blockchain': blockchain,
-        'asset-in': quote.sourceCurrency.toString(),
+        'asset-in': quote.fiatCurrency.toString(),
```

Comment:
wouldn't this change based on whether it's a buy or sell?

Reply:
fixed